frontend/src/components/cards/age/age.jsx

```diff
@@ -64,21 +65,21 @@ export class AgeCard extends React.PureComponent {
           label: {
             show: true,
             position: 'right',
-            formatter: (rawData) => {
-              const item = rawData
-              return ` ${item.data.value} (${item.data.percentage}%) `
+            formatter: rawData => {
+              const item = rawData;
+              return ` ${item.data.value} (${item.data.percentage}%) `;
             },
             color: 'black',
             fontWeight: 'bold'
           }
-        },
+        }
       ]
-    }
-  }
+    };
+  };
 
   render() {
     const { title, state } = this.props;
-    const { isLoaded, error } = state;
+    const { isLoaded, error, lastUpdatedOnString, stale } = state;
```

Comment:
nit
```suggestion
    const { isLoaded, error, lastUpdatedDate, stale } = state;
```

Reply:
In dashboard we send lastUpdatedOnString

Reply:
Ah right. We could alias, but works as it is 👍